### ISSUE TITLE
Mozilla Apps Enhancements and Firefox push notifications

### ISSUE THREAD

**Issue by barramaurizio** [javidominguez/MozillaScripts#13]
Hello and congratulations for your work!
But I would like to raise a problem:
as you well know, notifications in Firefox are not accessible with Nvda and your component allows them to be read only, without being able to interact with them.
Instead, this happens in Chrome and Edge, which can be managed directly from the Windows Notification Center, from which the respective sites for which I have activated the push notifications that interest me open.
Since this mode doesn't work with your addon, I thought that by entering via
about:config
in Firefox the option to send the Windows Notifications center, they could be opened, but this only happens for the first one and not always or sporadically.
So I restored the original option and at this point I have no hope that with his skill and his professional skills, he will be able to make Firefox alive, being able to open the notified sites.
Thank you so much and I really hope to see this feature soon.
Maurizio
